packages/javascript/bh-shared-ui/src/components/CreateUserForm/CreateUserForm.tsx

```diff
@@ -32,12 +32,12 @@ import {
 import React, { useEffect } from 'react';
```

Comment:
Im seeing some _smol_ formatting changes in here after running `prepare-for-codereview`